Engines/Wine/Plugins/DOS support/script.js

```diff
@@ -0,0 +1,102 @@
+include(["engines", "wine", "engine", "object"]);
+include(["utils", "functions", "filesystem", "files"]);
+
+Wine.prototype.dosbox = function () {
```

Comment:
The prototype itself is missing a comment.

Reply:
Done